sdk/ai/azure-ai-resources/azure/ai/resources/client/_ai_client.py

```diff
@@ -315,14 +315,14 @@ def build_index_on_cloud(
             IndexSource,
             IndexStore,
         )
-        from azure.ai.resources._index._embeddings.EmbeddingsContainer import from_uri
+        from azure.ai.resources._index._embeddings import EmbeddingsContainer
```

Comment:
Could you also apply this fix to this file: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ai/azure-ai-resources/azure/ai/resources/_index/_mlindex.py